External/Plugins/HaXeContext/Context.cs

```diff
@@ -388,6 +388,8 @@ public override void BuildClassPath()
             // add haxe libraries
             if (proj != null)
             {
+                proj.UpdateVars(false);
```

Comment:
FD definitely doesn't pick up on haxelib version changes right now when pressing "Refresh"  - for OpenFL projects at least.

The haxelibCache is indeed cleared when "Refresh" is pressed, but it's empty to begin with for those projects. It seems to be crucial that display is run again and that hxml is parsed for the new versions, which happens in `ExternalToolchain::UpdateProject()` (which my `UpdateVars()`-call leads to).